### DISCUSSION TITLE
qdmr v0.11.3 application settings dialog - unable to display

### DISCUSSION THREAD

**Discussion by knoxjl** [hmatuschek/qdmr#405] [General]
I can find no way to display application settings dialog on qdmr v0.11.3.
Installed on Windows 11 WSL2 Debian 12 (bookworm).
Manual does not indicate how to display dialog.
All other screens/dialogs appear to function as documented.

Please provide explanation how to display application settings dialog.

Thank you.

knoxjl@comcast.net

**Comment by allesand** [comment]
Hm, the "Settings" dialog (https://dm3mat.darc.de/qdmr/manual/ch02s11.html) is not displayed when you press the "gear" icon in the main window? To make sure: that toolbar is displayed (since it's detachable and this is the first report of qdmr running on WSL2, I think)?

![grafik](https://github.com/hmatuschek/qdmr/assets/123975169/18975105-e98c-4380-8717-a2516d88a5f4)

@knoxjl Have you tried to access your radio from qdmr running on WSL2, and does it work?

@hmatuschek There is no option for the settings dialog in the drop-down text menus, is there?

**Comment by knoxjl** [comment]
Thank you - “gear” displays dialog.

**Comment by allesand** [comment]
@knoxjl : perfect! When you tried, please report whether qdmr can successfully access the radio (and which one) when running on Windows with WSL2 - you seem to be the first to try that setup!

**Comment by knoxjl** [comment]
qdmr on WSL2 appears to work for
TYT MD-UV380
AnyTone 878UV II Plus

**Reply by hmatuschek** [reply]
Ok, thanks. Good to know.

**Reply by allesand** [reply]
@knoxjl : Very cool! Interesting that access to the radios is working "out of the box".
